locales/fr.json

```diff
@@ -0,0 +1,95 @@
+{
+  "welcome-back-title": "Bon retour dans Buttercup.",
+  "unlock-archive": "Déverrouillez une archive pour commencer ({os}).",
+  "welcome-title": "Bienvenue dans Buttercup.",
+  "welcome-caption": "Vous n’avez pas encore ajouté d’archive. Pourquoi ne pas en ajouter une ?",
+  "open-archive-file": "Ouvrir un ficier archive",
+  "new-archive-file": "Créer un fichier archive",
+  "connect-cloud-sources": "Connecter des sources sur le Cloud",
+  "add-archive": "Ajouter une archive",
+  "cancel": "Annule",
+  "go-back": "Retour en arrière",
+  "new-archive": "Nouvelle archive",
+  "open-archive": "Open Archive",
+  "open-in-buttercup": "Ouvrir dans Buttercup",
+  "master-password": "Mot de passe principal",
+  "confirm": "Confirmer",
+  "nevermind": "Pas grave…",
+  "password": "Mot de passe",
+  "language": "Langue",
+  "copy-to-clipboard": "Copier dans le presse-papier",
+  "edit": "Éditer",
+  "view": "Visualiser",
+  "window": "Fenêtre",
+  "help": "Aide",
+  "visit-our-website":"Visitez notre site",
+  "privacy-policy":"Politique de confidentialité",
+  "view-changelog-for-v":"Voir la liste des modifications de la v{version}",
+  "speech":"Aide vocale",
+  "condensed-sidebar":"Replier la colonne latérale",
+  "auto-hide-menubar":"Cacher automatiquement la barre de menu",
+  "new-group":"Nouveau groupe",
+  "title-asc":"Titre : Croissant",
+  "title-desc":"Titre : Décroissant",
+  "time-asc":"Heure : Croissant",
+  "time-desc":"Heure : Décroissant",
+  "add-group":"Ajouter un groupe",
+  "rename":"Renommer",
+  "move-to-root":"Déplacer vers la racine",
+  "move-to-group":"Déplacer vers un groupe",
+  "move-to-group-custom": "Déplacer vers {title}",
+  "move-to-trash":"Mettre à la corbeille",
+  "delete":"Supprimer",
+  "delete-permanently":"Suppression définitive",
+  "add-entry":"Ajouter un élément",
+  "search":"Chercher",
+  "select-or-create-an-entry":"Sélectionner ou créer un élément",
+  "save":"Enregistrer",
+  "empty-trash": "Vider la corbeille",
+  "empty-trash-question": "Êtes-vous sûr de vouloir vider cette corbeille ?",
```

Comment:
'la' instead of 'cette' would make more sense, as there's only 1 of it

Reply:
OK